src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -308,6 +308,14 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
         attachRef(current, finishedWork, instance);
         return;
       }
+      case HostContainer: {
+        const instance = finishedWork.stateNode;
+        if (instance.callbackList) {
+          const { callbackList } = instance;
```

Comment:
@gaearon spaces inside curly braces aren't needed?
I can see spaces inside curly braces in the ReactFiber codebase.

* https://github.com/facebook/react/blob/master/src/renderers/shared/fiber/ReactFiberCompleteWork.js#L126

Is there a rule for that?